From a72d8845545d6ac3b64b6fc48d0e2ada5750f6fe Mon Sep 17 00:00:00 2001
From: Takashi Kajinami <kajinamit@oss.nttdata.com>
Date: Fri, 27 Jun 2025 23:47:37 +0900
Subject: [PATCH] Replace deprecated assertItemsEqual

It has been provided by testtools to ease migration from python 2, but
was deprecated in 2.7.2[1] and will be removed in 2.8.0[2].

[1] https://github.com/testing-cabal/testtools/commit/e0d56b7ce65ae5b3d
[2] https://github.com/testing-cabal/testtools/commit/f01e86084e6a858d1

Change-Id: I86ae9c8daacccd09874f83d1142f02c80256031f
---
 os_client_config/tests/test_config.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/os_client_config/tests/test_config.py b/os_client_config/tests/test_config.py
index 04d4bf6..862e658 100644
--- a/os_client_config/tests/test_config.py
+++ b/os_client_config/tests/test_config.py
@@ -47,7 +47,7 @@ def test_get_all_clouds(self):
             '_test_cloud_regions'
         ]
         configured_clouds = [cloud.name for cloud in clouds]
-        self.assertItemsEqual(user_clouds, configured_clouds)
+        self.assertCountEqual(user_clouds, configured_clouds)
 
     def test_get_one_cloud(self):
         c = config.OpenStackConfig(
